### PR TITLE
Restyling clusterHelper.js

### PR DESCRIPTION
**Pull request by brdandu**
#### Problem
ClustersHelper.js is not restyled so it shows a large diff when making changes to it.

#### Change overview
Ran ./scripts/helpers/restyle-diff.sh on ClustersHelper.js

#### Testing
No tests were added for this.

### CODE REVIEW

File: src/app/zap-templates/common/ClustersHelper.js [hunk 1]
```diff
@@ -560,7 +560,7 @@ class ClusterStructUsage {
   }
 }
 
-Clusters._addUsedStructureNames = async function(clusterName, startType, allKnownStructs) {
+Clusters._addUsedStructureNames = async function (clusterName, startType, allKnownStructs) {
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/common/ClustersHelper.js:563]: I can't reproduce these changes by running `restyle_diff.sh` locally, so far.  What exact commands were run, starting with a clean master, to generate this diff?

Comment (andy31415) [reply src/app/zap-templates/common/ClustersHelper.js:563]: I believe js files are already restyled, however maybe only restyled when changed.

I also could not see this style change, only a change in items.forEach invocation in inlineStructItesms

**Review by andy31415 — CHANGES_REQUESTED** [review]
existing restyler seems to run different restyle rules.

How do we reproduce the same restyle deltas? Strangely enough, restyler ci check does not see things changed, so this seems to be using a stricter restyler somehow.

**Comment by andy31415** [comment]
My guess is that this was restyled with a different clang-format version than CI uses. It looks like our restyle.yaml uses clang-format for .js files.

**Comment by andy31415** [comment]
> My guess is that this was restyled with a different clang-format version than CI uses. It looks like our restyle.yaml uses clang-format for .js files.

I tried with clang-format 13 and still I do not see a similar delta.

**Pull request closed without merging**